Dockerfile.lint

```diff
@@ -0,0 +1,15 @@
+FROM registry.redhat.io/ubi8/ubi-minimal:8.7-1031 as builder
+
+RUN NODEJS_VERSION=16 && \
+    microdnf module disable nodejs && \
+    microdnf module enable nodejs:$NODEJS_VERSION && \
+    microdnf --nodocs install npm && \
+    microdnf clean all && \
+    rm -rf /mnt/rootfs/var/cache/* /mnt/rootfs/var/log/dnf* /mnt/rootfs/var/log/yum.*
+
+WORKDIR /usr/src/app
```

Comment:
Copying only the package json and installing dependencies before copying the files creates a container layer and caches the dependencies if the haven't changed:
```
COPY web/package*.json .
RUN npm ci
```